onnxruntime/test/providers/cpu/nn/qlinearconv_op_test.cc

```diff
@@ -548,6 +604,16 @@ TEST(QLinearConvTest, Conv2D_U8S8) {
   test.Run();
 }
 
+TEST(QLinearConvTest, Conv3D_U8S8) {
```

Comment:
can you add Conv1D cases while you're here?

Reply:
No problem!